current_pose_ros_service: True
goal_pose_ros_service: False
